### PR TITLE
use CoinGecko instead of CoinMarketcap for exchange rates

### PR DESCRIPTION
**Pull request by ayrat555**
CoinMarketcap deprecated and took offline its v1 API. They have
v2 API which has a free plan and requires API keys. But as part of
https://github.com/poanetwork/blockscout/issues/234 we already
implemented fetching from CoinGecko.

Now, coin id should be set for CoinGecko for the required coin.
```
config :explorer, Explorer.ExchangeRates.Source.CoinGecko,
  coin_id: System.get_env("COIN_GECKO_ID", "poa-network")
```
Or the `COIN_GECKO_ID` env var should be set. For example, COIN_GECKO_ID=poa-network

fixes https://github.com/poanetwork/blockscout/issues/2579

## Changelog

- use CoinGecko instead of CoinMarketcap for exchange rates

### CODE REVIEW

**Comment by ayrat555** [comment]
@vbaranov it was json decoding issue. fixed

**Comment by ayrat555** [comment]
@vbaranov 
1. It works for me. Along with `COIN_GECKO_ID`, `COIN` env var should be set
2. correct id can be found here https://api.coingecko.com/api/v3/coins/list

**Comment by ayrat555** [comment]
@vbaranov https://api.coingecko.com/api/v3/coins/poa-network is used for exchange data and market cap. Locally, I don't see this issue. How can I reproduce it?

File: docs/env-variables.md [hunk 1]
```diff
@@ -14,54 +14,55 @@ $ export NETWORK=POA
...
+| `ELIXIR_VERSION` | | Elixir version to install on the node before Blockscout deploy. | (empty) | all | | |
+| `BLOCK_TRANSFORMER` | | Transformer for blocks: base or clique. | base |  v1.3.4+ | | |
+| `GRAPHIQL_TRANSACTION` | | Default transaction in query to GraphiQL. | (empty) |  v1.2.0+ | :white_check_mark: | |
+| `FIRST_BLOCK` | | The block number, where indexing begins from. | 0 |  v1.3.8+ | | |
+| `LAST_BLOCK` | | The block number, where indexing stops. | (empty) | v2.0.3+ | | |
+| `TXS_COUNT_CACHE_PERIOD` | | Interval in seconds to restart the task, which calculates the total txs count. | 60 * 60 * 2 |  v1.3.9+ | | |
+| `ADDRESS_WITH_BALANCES` <br /> `_UPDATE_INTERVAL`|  | Interval in seconds to restart the task, which calculates addresses with balances. | 30 * 60 |  v1.3.9+ | | |
+| `LINK_TO_OTHER_EXPLORERS` | | true/false. If true, links to other explorers are added in the footer  | (empty)  |  v1.3.0+ | | |
+| `COINMARKETCAP_PAGES` | | the number of pages on coinmarketcap to list in order to find token's price  | 10 |  v1.3.10+ | | master |
```
Comment (vbaranov) [inline docs/env-variables.md]: @ayrat555 let's remove?

File: docs/env-variables.md [hunk 2]
```diff
@@ -14,54 +14,55 @@ $ export NETWORK=POA
...
+| `ETHEREUM_JSONRPC_WS_URL` | :white_check_mark: | The WebSockets RPC endpoint used to subscribe to the `newHeads` subscription alerting the indexer to fetch new blocks. | ws://localhost:8546 | all | | |
+| `NETWORK_PATH` | | Used to set a network path other than what is displayed in the root directory. An example would be to add /eth/mainnet/ to the root directory. | (empty) | all | | |
+| `SECRET_KEY_BASE` | :white_check_mark: | Use mix phx.gen.secret to generate a new Secret Key Base string to protect production assets. | (empty) | all | | |
+| `CHECK_ORIGIN` | | Used to check the origin of requests when the origin header is present. It defaults to false. In case of true, it will check against the host value. | false | all | | |
+| `PORT` | :white_check_mark: | Default port the application runs on is 4000 | 4000 | all | | |
+| `COIN` | :white_check_mark: | The coin here is checked via the Coinmarketcap API to obtain USD prices on graphs and other areas of the UI | POA | all | | |
```
Comment (vbaranov) [inline docs/env-variables.md]: @ayrat555 no more *CMC*

File: docs/env-variables.md (hunk 1 above)
Comment (ayrat555) [reply docs/env-variables.md]: I added deprecated in version - `master`.

File: docs/env-variables.md (hunk 2 above)
Comment (ayrat555) [reply docs/env-variables.md]: fixed

**Comment by vbaranov** [comment]
@ayrat555 

>Locally, I don't see this issue. How can I reproduce it?

I still this issue:

Screenshot 2019-08-28 at 20 24 19

Why is the difference? Just start this branch for POA chain instance

**Comment by ayrat555** [comment]
@vbaranov I fixed a market cap calculation

**Pull request merged**